types/d3-axis/tslint.json

```diff
@@ -1,7 +1,6 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "unified-signatures": false,
-        "no-unnecessary-generics": false
```

Comment:
This will need to be added back in, in light of the `scale<A ...>(): A` signature.